.github/workflows/ci-workflow.yml

```
@@ -47,6 +47,7 @@ jobs:
 
       - name: Cleanup the repository
         run: |
+          git config --global --add safe.directory /__w/app-xrp/app-xrp
```

Probably better version (used in other places):
```suggestion
          git config --global --add safe.directory "$GITHUB_WORKSPACE"
```